### PR TITLE
[next] Add tests for optional catch-all fallback true page params

### PR DESCRIPTION
**Pull request by ijjk**
This adds tests for optional catch-all `fallback: true` pages to ensure the params are normalized correctly. These should be failing initially until the PR with the normalizing fix in Next.js is released to canary. 

x-ref: https://github.com/vercel/next.js/issues/17220

### CODE REVIEW

**Comment by iwaki710** [comment]
希望

**Comment by iwaki710** [comment]
希望

**Comment by iwaki710** [comment]
希望

**Comment by iwaki710** [comment]
希望

**Comment by iwaki710** [comment]
希望

**Review by dstiers11 — APPROVED** [review]

**Review by Timer — DISMISSED** [review]

File: packages/now-next/test/fixtures/00-optional-fallback-revalidate/now.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (styfle) [inline packages/now-next/test/fixtures/00-optional-fallback-revalidate/now.json]: Let's start using `vercel.json` for new tests (the test harness should already support this I believe)

**Review by styfle — APPROVED** [review]

**Review by Timer — APPROVED** [review]

**Pull request merged**